src/components/SpaceDetailsPage/SpaceDetailsCard.jsx

```diff
@@ -222,7 +222,7 @@ const SpaceDetailCard = ({
         subheader={<Typography variant="body2">{category}</Typography>}
         classes={{ action: classes.headerAction }}
       />
-      <CardMedia image={imageUrl} className={classes.cardMedia} />
+      {imageUrl !== '' ? <CardMedia image={imageUrl} className={classes.cardMedia} /> : ''}
```

Comment:
We should probably use a fallback image here instead of an empty CardMedia. Can you add a todo here?

Reply:
Absolutely!